sqlmock_go18_19.go

```diff
@@ -0,0 +1,11 @@
+// +build go1.8,!go1.9
```

Comment:
I wanted to preserve the `CheckNamedValue` functionality for 1.8, but provide my own for 1.9+.  I wasn't sure if there was a better way to handle doing that though 😅